tools/modelgen/generators/templates/register.tmpl

```diff
@@ -0,0 +1,5 @@
+{{ range .Objects }}
+func Register{{ initialCap .Name }}Callbacks(handler {{ initialCap .Name }}Callbacks) {
+  objCallbackHandler.{{ initialCap .Name }}Cb = handler
```

Comment:
May be we should add a check for `handler` being nil and initialize the `<>Cb` to a default no-op handler in that case?

This will permit objmodel clients to just provide callback handlers just for objects they are interested in and rest of handlers will be set to default no-op handler.